stable/redis/values.yaml

```diff
@@ -64,6 +64,30 @@ networkPolicy:
   ##
   # allowExternal: true
 
+serviceAccount:
+  # Specifies whether a ServiceAccount should be created
+  create: false
+  # The name of the ServiceAccount to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name:
+
+rbac:
+  # Specifies whether RBAC resources should be created
+  create: false
```

Comment:
I would use `enabled` as we do for other components such as **livenessProbes**, **readinessProbes**, etc.

Reply:
Forget about this comment. As it's documented at https://github.com/kubernetes/helm/blob/master/docs/chart_best_practices/conventions.md, `create` is fine.